src/sonic-config-engine/minigraph.py

```diff
@@ -369,7 +411,7 @@ def parse_cpg(cpg, hname):
                         'keepalive': keepalive,
                         'nhopself': nhopself
                     }
-                else:
+                if start_router.lower() == hname.lower():
```

Comment:
can you explain the change here? why not elif?

Reply:
When parsing for specific asic name, we have to make sure that either start router or end router matches with the asic name because, in multiasic device Minigraph, Cpg will include information of all ASICs. 
Changed to "elif".